src/pages/library/common.js

```diff
@@ -85,11 +87,30 @@ const Sidebar = ({ onSectionFilter, onTagFilter, sections, selectedSections, sel
   ])
 }
 
+const reverseText = _.flow(_.split(''), _.reverse, _.join(''))
```

Comment:
Do we need to `_.split`?

```suggestion
const reverseText = _.flow(_.reverse, _.join(''))
```

Reply:
I didn't realize lodash `reverse` would automatically convert the string to an array - thanks!